src/ai-bundle/config/options.php

```diff
@@ -88,6 +88,7 @@
                     ->arrayNode('ollama')
                         ->children()
                             ->scalarNode('host_url')->defaultValue('http://127.0.0.1:11434')->end()
+                            ->scalarNode('cache')->end()
```

Comment:
We might end with the same cache repeated again and again in every platform.

Should we introduce a cache config key at a higher level ?

Reply:
Thanks